### ISSUE TITLE
FanControlWorker: Control unavailable

### ISSUE THREAD

**Issue by dafffooo** [tuxedocomputers/tuxedo-control-center#33]
Hi,
thank you for this software, it really enriches Tuxedo line. However I have a slight issue with tcc running on Tumbleweed 20200717 -- 5.7.7-1-default, regarding fan controls. I don't know if I need some additional package or have them to much perhaps.

(1) TCC shows fan and temp widgets blurred
(2) systemctl status tccd 
```
srp 20 22:52:37 linux-uoa7 systemd[1]: Started TUXEDO Control Center Service.
srp 20 22:52:37 linux-uoa7 tccd[5097]: Starting daemon v1.0.3 (node: v10.15.3 arch: x64)
srp 20 22:52:37 linux-uoa7 tccd[5097]: No tuxedo-cc-wmi found
srp 20 22:52:37 linux-uoa7 tccd[5097]: Set display brightness to 50% (12000) on intel_backlight
srp 20 22:52:37 linux-uoa7 tccd[5097]: Daemon started
srp 20 22:52:38 linux-uoa7 tccd[5097]: FanControlWorker: Control unavailable
```

```
tlp-rdw-1.3.1-1.1.noarch
tlp-1.3.1-1.1.noarch
tuxedo-cc-wmi-0.1.4-1.noarch
tuxedofancontrol-0.1.10-1.noarch
tuxedo-keyboard-2.0.5-1.noarch
tuxedo-one-wallpapers-4.0.4-lp151.2.1.x86_64
tuxedo-control-center-1.0.3-1.x86_64
```

Kind regards,
Davor V.

**Comment by tuxedoxt** [comment]
Hello,

most likely tuxedo-cc-wmi is not loaded.

You can check the install status with `dkms status`. If it says installed for the kernel you are using, linux should load it automatically. If not you could try to install it with `sudo dkms install -m tuxedo-cc-wmi -v 0.1.4`, then reboot or `sudo modprobe tuxedo-cc-wmi`.

**Comment by dafffooo** [comment]
Hi,
thank you,  `dkms install` solved it.

Only thing I still see is "old" version on wmi that dkms status complaints about.

```
# dkms status
realtek-clevo-pin-fix, 1.0.0: added
Error! Could not locate dkms.conf file.
File: /var/lib/dkms/tuxedo-cc-wmi/0.1.3/source/dkms.conf does not exist.
```
I've checked 
` /lib/modules/5.7.7-1-default/kernel/lib/tuxedo_cc_wmi.ko.xz`
and it's a proper version
```
# strings -a tuxedo_cc_wmi.ko | grep "/var/lib/dkms/tuxedo-cc-wmi"
/var/lib/dkms/tuxedo-cc-wmi/0.1.4/build/./src/tuxedo_cc_wmi.c
/var/lib/dkms/tuxedo-cc-wmi/0.1.4/build/./src/tongfang_wmi.h
/var/lib/dkms/tuxedo-cc-wmi/0.1.4/build/./src/clevo_wmi.h
```
Nevertheless, issue is solved.

Regards,
Davor

**Comment by tuxedoxt** [comment]
Glad to hear it works!

Question is how it ended up in this state since it seems you are using our packages. Will keep it in mind.

To clean up the old version you can try `dkms remove -m tuxedo-cc-wmi -v 0.1.3 --all` or if that doesn't help (remove 0.1.3 artifacts from `/usr/src` and `/var/lib/dkms/tuxedo-cc-wmi`).
